### ISSUE TITLE
Grid layout for Gallery Article Type

### ISSUE THREAD

**Issue by sonvnn** [templaza/astroid-framework#22]
Display gallery as Grid Columns:
- Grid column
- Masonry
- Horizontal Masonry

**Comment by nidzo80** [comment]
Can't wait to see this feature. Would be good to have drag and drop field for multiple image upload!

**Comment by sonvnn** [comment]
> Can't wait to see this feature. Would be good to have drag and drop field for multiple image upload!

Yes, It will be released in v2.5.11. I am working on Jollyany, I will soon back to Astroid and release new version.
Thanks for your support :)

**Comment by sonvnn** [comment]
Hello @Joomlaplates, 

An issue happens when I code this feature. That is attribs field is not enough to save data. Now it's varchar(5120). So it will overflow when we create gallery with 8+ images. May you give me some advise about it? 

Anyone who have any suggestions are welcome :). I asked Joomla core team about it, but they said "They will not update it to longtext". I am thinking about store content in a new table, but it's the last way if we can not find any solution.

Best Regards,
Sonny

**Comment by djayko** [comment]
Hello @sonvnn,
only briefly beforhand, I'm not a developer, but I've an advanced basic knowledge of programming. :-)
My intenion is, giving an impuls.

Here it is:
Is it possible to create a json file for the grid-gallery details an store that file "anywhere" and the attribs field only stores the link to that file?
Due to the fact that it's almost static content I guess it would be ok in combination of caching.

Just as basic idea without knowing every dependency or (dis)advantages.

best regards,
Frank

**Comment by nidzo80** [comment]
Any news about this feature?

**Comment by sonvnn** [comment]
Hi @nidzo80 @djayko 

Thanks for your feedback.. I will consider about this after release Color Mode feature
Please wait for update.

Best Regards,
Sonny

**Comment by Chacapamac** [comment]
Grid layout for Gallery Article Type — 
That will be a great addition and all great article types should be part of the documentation.

**Comment by felixkat** [comment]
Closing as I believe this feature has now been implemented with Grid and Image Group.

Please create a new issue if there is anything missing that I have overlooked.
